trustpinning/trustpin.go

```diff
@@ -62,6 +62,14 @@ func NewTrustPinChecker(trustPinConfig TrustPinConfig, gun string) (CertChecker,
 	if !trustPinConfig.DisableTOFU {
 		return t.tofusCheck, nil
 	}
+
+	// If we already have previous trusted root data for this GUN, disabling TOFUs should not change trust
+	// At this point, we assume that we have already trusted previous root data and so will not perform any
+	// additional trust pinning checks on this new certificate
+	if !firstBootstrap {
+		logrus.Debug("TOFU is disabled but previous root data exists, skipping disable TOFU restriction")
```

Comment:
Nit:  I'm wondering if this log message might be confusing, since it's not exactly doing a TOFU check, since it's not first use.  It's just that no other trust pinning has been specified.  In which case maybe this check can just be combined with the previous conditional?